cli_tools/common/distro/distro.go

```diff
@@ -0,0 +1,179 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package distro
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+// Release encapsulates product and version information
+// about a Linux or Windows release.
```

Comment:
Why only Linux and Windows? If, in the future, we have other OS's, would they be supproted?

Reply:
talked offline